Packs/ArrayToCSV/pack_metadata.json

```diff
@@ -1,8 +1,11 @@
 {
     "name": "ArrayToCSV",
     "description": "Converts a simple Array into a textual comma separated string",
+    "name": "ArrayToCSV (Deprecated)",
+    "description": "Deprecated. Use CommonScripts instead.",
```

Comment:
Why the double description? 
Just update the description that exists and remove the added one